config/locales/en.yml

```diff
@@ -36,3 +36,15 @@ en:
   premium_reminder:
     premium_reminder: 'Premium Reminder'
     do_not_disturb: 'Do Not Disturb'
+  devise:
+    registrations:
+      address_fields:
+        address: 'Address'
+        same_as_present: 'Same as present'
+        present_address: 'Present Address'
+  customers:
+    new:
```

Comment:
@VaibhaviJoshi-BTC :This should not be new because the fields under this might be needed for some other action
. Use a diff key like 'field'